opentech/public/news/templates/news/news_index.html

```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %}
 {% load wagtailcore_tags wagtailimages_tags static markdown_tags bleach_tags %}
+{% block feedlinks %}<link rel="alternate" type="application/rss+xml" title="RSS" href="{{ request.site.root_url }}{{ request.path }}feed/">{% endblock %}
```

Comment:
@frjo We can use [url](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#url) template tag to get URL. This way we can avoid hard-coded values.

Reply:
A lot nicer now, thanks.